rules.md

```diff
@@ -451,31 +450,31 @@ In the Debug Room, at (009,032), there is an event with basic code for equipping
 
 Make sure to thoroughly playtest all actions (sitting on benches, climbing ladders, entering and exiting boats) and gestures to make sure implementation was done correctly! Include the pseudo's equip location in your changelog on submission.
 
-## Contributing Visages
+## Contributing Masks
```

Comment:
The title of the section was changed to Contributing Masks, but the link to the section itself was not changed.